pkg/util/test/util.go

```diff
@@ -1480,7 +1480,7 @@ func ValidateStorkEnabled(pxImageList map[string]string, cluster *corev1.Storage
 		}
 
 		opVersion, _ := GetPxOperatorVersion()
-		if opVersion.LessThanOrEqual(opVer1_10_1) {
+		if opVersion.LessThan(minOpVersionForKubeSchedConfig) {
 			if kubeVersion != nil && kubeVersion.GreaterThanOrEqual(K8sVer1_22) {
 				// Image tag for stork-scheduler is hardcoded to v1.21.4 for clusters 1.22 and up for Operator version 1.10.1 and below
```

Comment:
comments need update as well

Reply:
since the change is going in 1.10.2 and for op version 1.10.1 and below we have to use the old validation logic, this comment is still true. Do you still think I should change the wordings here?